src/vmm/src/rpc_interface.rs

```diff
@@ -387,20 +402,42 @@ impl RuntimeApiController {
 
     /// Pauses the microVM by pausing the vCPUs.
     pub fn pause(&mut self) -> ActionResult {
+        let pause_start_us = utils::time::get_time_us(utils::time::ClockType::Monotonic);
+
         self.vmm
             .lock()
             .expect("Poisoned lock")
             .pause_vcpus()
-            .map_err(VmmActionError::InternalVmm)
+            .map_err(VmmActionError::InternalVmm)?;
+
+        update_metric_with_elapsed_time(
+            &METRICS.latencies_us.vmm_pause_vm,
+            pause_start_us,
+            "pausing the microVM",
```

Comment:
I think it would be much better like this:
```suggestion
            "pause_vcpus()",
```